### PR TITLE
feat: add warningsWhitelist to entryConfig.json

### PR DESCRIPTION
**Pull request by teppeis**

### CODE REVIEW

File: src/entryconfig.ts [hunk 1]
```diff
@@ -91,6 +98,11 @@ export async function loadEntryConfig(
       mod.inputs = mod.inputs.map(input => path.resolve(basedir, input));
     });
   }
+  if (entryConfig.warningsWhitelist) {
+    Object.values(entryConfig.warningsWhitelist).forEach(item => {
```
Comment (koba04) [inline src/entryconfig.ts]: Why is this `Object.values` required? Is there any problem just to use `entryConfig.warningsWhitelist.forEach`?

Comment (teppeis) [reply src/entryconfig.ts]: @koba04 nice catch!

Comment (teppeis) [reply src/entryconfig.ts]: @koba04 fixed! https://github.com/teppeis/duck/commit/1f05ed90527148037c2123a61b36500ab5e85555

Comment (koba04) [reply src/entryconfig.ts]: Thanks!

**Pull request merged**